production/helm/loki/README.md

```diff
@@ -1,69 +1,406 @@
-# ⚠️  DEPRECATED - Loki Helm Chart
+# loki-simple-scalable
 
-This chart was moved to <https://github.com/grafana/helm-charts>.
+![Version: 1.8.4](https://img.shields.io/badge/Version-1.8.4-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 2.6.1](https://img.shields.io/badge/AppVersion-2.6.1-informational?style=flat-square)
 
-## Prerequisites
+Helm chart for Grafana Loki in simple, scalable mode
 
-Make sure you have Helm [installed](https://helm.sh/docs/using_helm/#installing-helm) and
-[deployed](https://helm.sh/docs/using_helm/#installing-tiller) to your cluster. Then add
-Loki's chart repository to Helm:
+## Source Code
 
-```bash
-$ helm repo add loki https://grafana.github.io/loki/charts
-```
+* <https://github.com/grafana/loki>
+* <https://grafana.com/oss/loki/>
+* <https://grafana.com/docs/loki/latest/>
 
-You can update the chart repository by running:
+## Requirements
 
-```bash
-$ helm repo update
-```
+| Repository | Name | Version |
+|------------|------|---------|
+| https://grafana.github.io/helm-charts | grafana-agent-operator(grafana-agent-operator) | 0.2.2 |
+| https://helm.min.io/ | minio(minio) | 8.0.9 |
+
+## Chart Repo
 
-## Deploy Loki only
+Add the following repo to use the chart:
 
-```bash
-$ helm upgrade --install loki loki/loki
+```sh
+helm repo add grafana https://grafana.github.io/helm-charts
 ```
 
-## Run Loki behind https ingress
+## Values
```

Comment:
I find these long READMEs often overwhelming. What do you think about linking to the documentation instead?

Reply:
I think we need to keep the values because of the auto-generation, but I'd be open to moving them to the bottom of the file.